libs/common-types/src/milestone_origin.rs

```diff
@@ -47,7 +47,7 @@ pub enum TreasuryOriginError {
     Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
 )]
 pub enum FundingType {
-    Proposal,
+    Project,
```

Comment:
I initially was thinking that this woud require a migration, buut since the index is the same we are all good :)